deps/CMakeLists.txt

```diff
@@ -14,17 +10,9 @@ function(mapbox_base_add_library name include_path)
         ${include_path}
     )
 
-    target_link_libraries(mapbox-base INTERFACE mapbox-base-${name})
+    # target_link_libraries(mapbox-base INTERFACE mapbox-base-${name})
```

Comment:
We still need this for this transitional period, right?

Reply:
yes

Reply:
Then we need to un-comment this.

Reply:
Comment outdated: https://github.com/mapbox/mapbox-base/blob/kdn-dev/deps/CMakeLists.txt#L13